app/controllers/concerns/set_for_end_of_month.rb

```diff
@@ -3,7 +3,17 @@ def set_for_end_of_month
     # Defaults to current month if it's the last week of the month.
```

Comment:
The `.present?` seems to fix the issue, why are we adding the Sentry information?

Reply:
I'm not sure that is the issue here...I want to have more context if this error (or others) show up in Sentry.